scripts/build/store-azp-variables.ps1

```diff
@@ -0,0 +1,15 @@
+# Read the project version from the Version.props file
+$VersionFilePath = "$env:BUILD_SOURCESDIRECTORY\scripts\version\Version.props"
+Write-Host "Reading the Sonar project version from '${VersionFilePath}' ..."
+
+[xml]$VersionProps = Get-Content "$VersionFilePath"
+$Version = $VersionProps.Project.PropertyGroup.MainVersion
+Write-Host "Sonar project version is '${Version}'"
+
+# Save variables to files so they can be used by other tasks and stages
+$Dir = "$env:AGENT_BUILDDIRECTORY\azp-variables"
+New-Item -ItemType Directory -Path $Dir | Out-Null
+
+$File = "${Dir}\SONAR_PROJECT_VERSION"
+Write-Host "Writing '${Version}' to ${File}"
+Set-Content -Path $File -Value $Version
```

Comment:
Intention: Each variable to have it's own file for simple reading. These 3 lines would repeat (or be moved in a function in the future)

Reply:
why would the lines repeat?

Reply:
Only in case we add more variables in the future. Lines would repeat for 2nd variable.